app/about/page.module.css

```
@@ -0,0 +1,143 @@
+.container {
+  background-color: rgb(var(--background-end-rgb));
+  margin: 0 auto;
+
+  --h1-size: 30px;
+  --h1-media: 6.3vw;
+}
+
+.title {
+  display: flex;
+  justify-content: center;
+}
+
+.title h1 {
+  font-family: var(--font-moos);
+  letter-spacing: -0.25vw;
+  font-size: var(--h1-size);
+  font-weight: 200;
+  padding: 0 1.5rem;
+  padding-top: 3rem;
+  padding-bottom: 1.5rem;
```

you can use shorthand for this
```suggestion
  padding: 3rem 1.5rem 1.5rem;
```